### ISSUE TITLE
Running Apache Benchmark raises Mach port errors.

### ISSUE THREAD

**Issue by beckyconning** [spajak/cef-pdf#10]
[0818/155022.479392:ERROR:node_controller.cc(1258)] Error on receiving Mach ports 9A3CE724EE0F1F50.8B6639709E110ED7. Dropping message.
[0818/155022.480067:ERROR:node_controller.cc(1258)] Error on receiving Mach ports 9A3CE724EE0F1F50.8B6639709E110ED7. Dropping message.
[0818/155022.482067:ERROR:node_controller.cc(1258)] Error on receiving Mach ports 9A3CE724EE0F1F50.8B6639709E110ED7. Dropping message.
[0818/155022.495248:ERROR:node_controller.cc(1258)] Error on receiving Mach ports D7544DA778CC245.E7F190FF8AEC0953. Dropping message.
[0818/155023.489002:ERROR:node_controller.cc(1258)] Error on receiving Mach ports F6E274AE0B134AD3.784323643B9E5F17. Dropping message.                                                 
[0818/155023.489103:ERROR:node_controller.cc(1258)] Error on receiving Mach ports F6E274AE0B134AD3.784323643B9E5F17. Dropping message.                                    
[0818/155023.489145:ERROR:node_controller.cc(1258)] Error on receiving Mach ports F6E274AE0B134AD3.784323643B9E5F17. Dropping message.                                               
[0818/155023.489191:ERROR:node_controller.cc(1258)] Error on receiving Mach ports 9CEE88C2FC00399B.6E7EE136C68DE6D5. Dropping message.                                                 
[0818/155023.489227:ERROR:node_controller.cc(1258)] Error on receiving Mach ports 9CEE88C2FC00399B.6E7EE136C68DE6D5. Dropping message. 

Not sure if this actually has any negative effects?

**Comment by beckyconning** [comment]
This happens on devel branch and on v0.3.1

**Comment by spajak** [comment]
What OS?

**Comment by beckyconning** [comment]
MacOS

**Comment by beckyconning** [comment]
Might be that it needs to be codesigned? Thats just a guess based on that mach ports don't work in gdb without code signing.

**Comment by beckyconning** [comment]
Yeah codesigning fixes this. E.g. `codesign --deep -s cef-pdf-cert ./src/Release/cef-pdf.app/Contents/MacOS/cef-pdf`

**Comment by spajak** [comment]
Couldn't reproduce it on `devel` branch

**Comment by spajak** [comment]
Never heard about `codesign`

**Comment by beckyconning** [comment]
what version of macos are you using? have you disabled the gatekeeper?

**Comment by spajak** [comment]
El Capitan, never heard about gatekeeper :) If it's something required maybe I should include it in cef-pdf releases

**Comment by beckyconning** [comment]
Its worth doing but you will need a verified certificate.

**Comment by beckyconning** [comment]
https://developer.apple.com/support/certificates/

**Comment by beckyconning** [comment]
If you need help with money for the certificate just let me know and I’ll PayPal you some.
